README.md

```diff
@@ -36,339 +34,133 @@
-
-The [Semgrep Registry](https://semgrep.dev/r) contains rules for many programming errors, including security issues and correctness bugs. Security rules are annotated with CWE and OWASP metadata when applicable. OWASP rule coverage per language is displayed below.
-
-<p align="center">
-    <img width="600" src="https://web-assets.r2c.dev/semgrep-rules-owasp-coverage-20200520.png" style="max-width:100%;" />
-</p>
-
-You can use pre-built [Rule Packs](https://semgrep.dev/packs), that contain sets of rules grouped by language and/or framework:
-
-```bash
-$ semgrep --config=https://semgrep.dev/c/p/java
-$ semgrep --config=https://semgrep.dev/c/p/python
-$ semgrep --config=https://semgrep.dev/c/p/golang
-$ semgrep --config=https://semgrep.dev/c/p/javascript
-...
 ```
-
-Or you can run all of Semgrep's default rules for all languages as appropriate (note: each rule says what language it's for, so Semgrep won't try to run a Python rule on Java code).
-
-```bash
-$ semgrep --config=r2c
-```
-
-You can also run a specific rule or group of rules:
-
-```bash
-# Run a specific rule
-$ semgrep --config=https://semgrep.dev/c/r/java.spring.security.audit.cookie-missing-samesite
-
-# Run a set of rules
-$ semgrep --config=https://semgrep.dev/c/r/java.spring.security
+$ semgrep --help
 ```
 
-All public Semgrep rules can be viewed on the [Registry](https://semgrep.dev/r), which pulls the rules from YAML files defined in the [semgrep-rules](https://github.com/returntocorp/semgrep-rules) GitHub repo.
-
-Here are some sample vulnerable repos to test on:
-* Django: [lets-be-bad-guys](https://github.com/mpirnat/lets-be-bad-guys), [django.nV](https://github.com/nVisium/django.nV)
-* Flask: [Vulnerable-Flask-App](https://github.com/we45/Vulnerable-Flask-App)
-* Java: [WebGoat](https://github.com/WebGoat/WebGoat), [OWASP Benchmark](https://github.com/OWASP/Benchmark)
-* NodeJS: [OWASP Juice Shop](https://github.com/bkimminich/juice-shop), [DevSlop Pixi](https://github.com/DevSlop/Pixi)
-* Golang: [GoVWA](https://github.com/0c34/govwa)
-
-### Writing Custom Rules
-
-One of the strengths of Semgrep is how easy it is to write rules.
+Once installed, Semgrep can be run with single rule patterns or entire rule packs:
 
-This makes it possible to:
-* Quickly port rules from other tools.
-* Think of an interesting code pattern, and then find instances of it in your
-    code.
-* Find code base or org-specific bugs and antipatterns - things that built-in
-    checks for existing tools won't find because they're unique to you.
-* and more!
+```sh
+# Check for Python == where the left and right hand sides are the same (often a bug)
+$ semgrep -e `$X==$X` --lang=py path/to/src
 
-#### Simple Rules
-
-For iterating on simple patterns, you can use the `--lang` and `--pattern`
-flags.
-
-```bash
-$ semgrep --lang javascript --pattern 'eval(...)' path/to/file.js
+# Run the default rule pack with rules for many languages
+$ semgrep --config=default path/to/src
```

Comment:
What does the default pack run? Should we recommend https://semgrep.dev/p/r2c-CI or https://semgrep.dev/p/r2c-security-audit?

Reply:
Let's do r2c-CI. Updated.